Sources/VideoIO/Camera.swift

```diff
@@ -159,6 +159,17 @@ public class Camera {
         self.configurator.videoDeviceConfigurator(self, device)
         device.unlockForConfiguration()
     }
+  
+    public func removeVideoCaptureDevice() {
+        self.captureSession.beginConfiguration()
+
+        if let currentVideoDeviceInput = self.videoDeviceInput {
```

Comment:
It's better to use `guard let` to only call the `beginConfiguration`/`commitConfiguration` when we do have a video device input.